tests/Kron.py

```diff
@@ -130,36 +228,35 @@ def measureKron(self, objImg, xcen, ycen, nsigma, kfac, nIterForRadius):
         #
         # Now measure things
         #
-        msConfig = measAlg.SourceMeasurementConfig()
-        if False:                       # requires #2546
-            msConfig.centroider = None
-            msConfig.slots.centroid = None
-
-        msConfig.algorithms.names.add("flux.kron")
-        msConfig.algorithms.names.remove("correctfluxes")
-        msConfig.algorithms["flux.kron"].nSigmaForRadius = nsigma
-        msConfig.algorithms["flux.kron"].nIterForRadius = nIterForRadius
-        msConfig.algorithms["flux.kron"].nRadiusForFlux = kfac
-        msConfig.algorithms["flux.kron"].enforceMinimumRadius = False
+        msConfig = measBase.SingleFrameMeasurementConfig()
+        msConfig.algorithms.names = ["base_SdssCentroid", "base_SdssShape", "extensions_photometryKron_KronFlux"]
+        msConfig.slots.centroid = "base_SdssCentroid"
+        msConfig.slots.shape = "base_SdssShape"
+        msConfig.slots.apFlux = "extensions_photometryKron_KronFlux"
+        msConfig.slots.modelFlux = None
+        msConfig.slots.psfFlux = None
+        msConfig.slots.instFlux = None
+        #msConfig.algorithms.names.remove("correctfluxes")
```

Comment:
Why is this here? (Probably because you want to add it back when you get to the FIXME above, but I think this isn't the right way to handle that).